pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css

```diff
@@ -5542,3 +5534,11 @@ treecontrol.tree-classic li .tree-selected {
 #sidepanel #sidepaneltabset > div[tabid=generaltab] > span{
   max-width:62px;
 }
+
```

Comment:
i don't see a change other than moving these lines down in the file... am i blind?